### PR TITLE
BVAL-647 Relax a few tck-testable assertions

### PR DESCRIPTION
**Pull request by gsmet**
* https://hibernate.atlassian.net/browse/BVAL-647

### CODE REVIEW

File: sources/constraint-declaration-validation.asciidoc [hunk 1]
```diff
@@ -816,16 +816,13 @@ private Map<String, @NotEmpty List<@ValidAddress Address>> addressesByType;
 
 In such case multiple value extractors will be invoked.
 
-[tck-testable]
```
Comment (gunnarmorling) [inline sources/constraint-declaration-validation.asciidoc]: Why is it that this one went away?

File: sources/constraint-declaration-validation.asciidoc [hunk 2]
```diff
@@ -816,16 +816,13 @@ private Map<String, @NotEmpty List<@ValidAddress Address>> addressesByType;
...
---
 In the example above,
 
 * the extractor for `Map` values will be invoked to obtain all map values (lists of `Address`)
 * for each extracted list of addresses, the `@NotEmpty` constraint will be validated and the extractor for `List` elements will be invoked, providing the `Address` objects from each list in the map
 * the `@ValidAddress` constraint will be applied to all elements of all lists stored in the map
---
 
-[tck-testable]#Container element constraints are not supported on wildcard type arguments (with or without bounds), i.e. the following usage is unsupported#:
+[tck-not-testable]#Container element constraints are not supported on wildcard type arguments (with or without bounds), i.e. the following usage is unsupported#:
```
Comment (gunnarmorling) [inline sources/constraint-declaration-validation.asciidoc]: This is alright.

File: sources/constraint-declaration-validation.asciidoc (hunk 1 above)
Comment (gsmet) [inline sources/constraint-declaration-validation.asciidoc]: It's the explanation of an example. We have other tests covering this and we usually don't implement all the examples as TCK tests.

Comment (gunnarmorling) [inline sources/constraint-declaration-validation.asciidoc]: Ok.

**Comment by gunnarmorling** [comment]
Thanks, @gsmet.

**Pull request merged**